examples/zero-tasks.rs

```diff
@@ -2,8 +2,11 @@
 #![deny(unsafe_code)]
 #![deny(warnings)]
 #![no_std]
+#![no_main]
 
+extern crate cortex_m_rt_macros;
```

Comment:
That would be better to depend on cortex_m_rt as you already need it on your Cargo.toml to use the rt feature.

Reply:
I think the better solution would be to reexport the needed macro directly in cortex-m-rtfm to not depend on extern crate by the user.

Reply:
I think you're right.

Could you check 0054ef7c387c911c168f103ba59ae3d4751a2201, please?

Reply:
Great! https://github.com/TeXitoi/rusty-clock/commit/1f10a38bc173f3661f6545afba71741dfd2b62c6